gems/pending/MiqVm/MiqVm.rb

```diff
@@ -300,7 +306,7 @@ def format(event)
     puts "\t#{rt.guestOS}"
     if rt.guestOS == "Linux"
       puts "\n\t\t*** /etc/fstab contents:"
-      rt.fileOpen("/etc/fstab", &:read).each_line do |fstl|
+      rt.fileOpen("/etc/fstab") { |fo| fo.read }.each_line do |fstl|
```

Comment:
Oh.  Possibly this was a rebase conflict in which I chose the wrong side of the diff - because rest assured I wouldn't have come up with that line on my own... its probably the old code.  I'll change it back.